test/suites/common/run-test.yaml

```diff
@@ -20,13 +20,21 @@ spec:
   - name: checkout-repository
     image: alpine/git
     workingDir: $(workspaces.ws.path)
+    resources:
```

Comment:
Can we use a limit range in this namespace to apply this by default to all pods?

Reply:
Neat idea, unfortunately it doesn't work because Tekton applies a tiny resource request when you don't specify a resource request:

```
    resources:
      requests:
        cpu: 41m
        ephemeral-storage: "0"
        memory: "44739242" # 42 MiB
```